### ISSUE TITLE
Quarantine permissions

### ISSUE THREAD

**Issue by pawp81** [MicrosoftDocs/microsoft-365-docs#909]
Could you please document what minimum permissions are required to access emails and files in the Quarantine?

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 5b34b5c8-b67e-72f2-0eba-d98b760f1d16
* Version Independent ID: adcd41fb-ffd1-f6d0-96ad-0036c0a677a7
* Content: [Permissions in the Office 365 Security & Compliance Center](https://docs.microsoft.com/en-us/microsoft-365/security/office-365-security/permissions-in-the-security-and-compliance-center)
* Content Source: [microsoft-365/security/office-365-security/permissions-in-the-security-and-compliance-center.md](https://github.com/MicrosoftDocs/microsoft-365-docs/blob/public/microsoft-365/security/office-365-security/permissions-in-the-security-and-compliance-center.md)
* Service: **o365-seccomp**
* GitHub Login: @chrisda
* Microsoft Alias: **chrisda**

**Comment by chrisda** [comment]
@pawp81, the topic that's the focus of this issue really isn't the best place to talk about that. But, I can see why you landed here, because the topic where we should talk about permissions for managing quarantine [Manage quarantined messages and files as an administrator in Office 365](https://docs.microsoft.com/microsoft-365/security/office-365-security/manage-quarantined-messages-and-files) tells you to see this topic!

Long story short, quarantine permissions are controlled by Exchange Online role groups, even though quarantine management is in Security & Compliance Center (confusing, I know).

I updated the quarantine topic with information about the required permissions, and a link for how to add people to role groups in Exchange Online.

Thanks for the feedback.
